test/services/postcard.spec.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+var expect  = require('chai').expect;
+var Angular = require('angular');
+var Sinon   = require('sinon');
```

Comment:
nit: add a newline here. we separate the packages we require from the local files we require